tutorials/viewer.md

```diff
@@ -1,58 +1,296 @@
-# Napari Viewer Tutorial
+# napari viewer tutorial
 
-Welcome to Napari Viewer Tutorial. In this tutorial we will be 
-covering listed topics about the viewer:
+Welcome to the tutorial on the **napari** viewer!
 
-- Starting the viewer
-- Changing between themes of viewer
-- Reordering layers on the viewer
-- Custom keybinding 
+This tutorial assumes you have already installed **napari** and know how to launch the viewer. For help with installation see our [installation](installation.md) tutorial. For help getting started with the viewer see our [getting started](getting_started.md) tutorial.
 
-### Starting the viewer
+This tutorial will teach you about the **napari** viewer, including how to use it on your screen and how the data within it is organised. At the end of the tutorial you should understand the both the layout of the viewer on the screen and the data inside of it.
```

Comment:
Are we standardizing on US spelling? If so, organised -> organized

Reply:
i don't have strong feelings, i go back in forth in my head without realizing, i fixed this